packages/components/src/components/SquiggleViewer/VariableBox.tsx

```diff
@@ -68,14 +74,19 @@ export const VariableBox: React.FC<VariableBoxProps> = ({
             <span className="text-stone-800 font-mono text-sm">{name}</span>
           </div>
           <div className="inline-flex space-x-1">
-            {!settings.collapsed && (
+            {Boolean(!settings.collapsed && location.path.items.length) && (
```

Comment:
Just curious - what's the benefit of the Boolean() here? For readability?

Reply:
It's not necessary, but I was overly paranoid about rendering 0. `false && 0 && <...>` would evaluate to `false` and not render anything, but `0 && <...>` would render to `0`. If you switch the order in the condition, it would also render `0`.

That's also why I sometimes use `foo ? <...> : null` instead of the shorter `foo && <...>`. I switchted to the latter style recently when I know for sure that the left side is boolean, but because of `.length` I wanted to be extra sure there's no bug here.

Reply:
Interesting. I sometimes would do ``!!foo`` for a similar purpose.